specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/examples/AcceptTransfer.json

```diff
@@ -21,21 +21,13 @@
   "responses": {
     "200": {
       "body": {
-        "id": "/providers/Microsoft.Billing/transfers/transferName",
-        "type": "Microsoft.Billing/transfers",
-        "name": "transferName",
         "properties": {
-          "creationTime": "10/10/2018 9:40:01 PM",
-          "expirationTime": "11/5/2018 4:47:08 AM",
-          "allowedSubscriptionType": [
-            "DevTestAzureSubscription",
-            "StandardAzureSubscription",
-            "AzureReservation"
-          ],
+          "creationTime": "2018-10-10T17:32:28Z",
+          "expirationTime": "2018-11-05T17:32:28Z",
           "transferStatus": "InProgress",
           "recipientEmailId": "user@contoso.com",
           "initiatorEmailId": "user@contoso.com",
-          "lastModifiedTime": "10/10/2018 9:45:01 PM",
+          "lastModifiedTime": "2018-10-10T17:32:28Z",
```

Comment:
Please make sure any changes made in examples, are reflecting what the service actually receives and responds with. These will feed documentation for customers.

Reply:
yes. We will be doing a more detailed scrubbing of the examples. This is just a first stab at cleaning it.